facebook-commerce.php

```diff
@@ -1428,9 +1430,25 @@ function update_product_group( $woo_product ) {
 			return;
 		}
 
-		$product_group_data = array(
+		// figure out the matching default variation
+		$default_product_fbid  = null;
+		$woo_default_variation = $this->get_product_group_default_variation( $woo_product );
+
+		if ( $woo_default_variation ) {
+			$default_product_fbid = $this->get_product_fbid(
+				self::FB_PRODUCT_ITEM_ID,
+				$woo_default_variation['variation_id'],
+				$woo_default_variation
```

Comment:
@N-Molham looks like `$woo_default_variation` is an array now but `get_product_fbid()` expects a `WC_Facebook_Product` in the third argument. Perhaps we don't need to pass the third argument and should allow `get_product_fbid()` to create the instance instead?

```suggestion
				$woo_default_variation['variation_id']
```

Reply:
Yeah, you are right ... the `get_product_fbid()` take care of that if the 3rd argument is `null`